packages/eslint-plugin-choreography-ts/tests/fixtures/tsconfig.json

```diff
@@ -0,0 +1,6 @@
+{
```

Comment:
Two questions:

1. Why do we need these empty files?
2. Why do we need a custom tsconfig?

Reply:
We need these so that typescript-eslint can use type information in rule testing as stated here: https://typescript-eslint.io/packages/rule-tester/#type-aware-testing

Reply:
Thanks for your explanation!